cvmfs/util/posix.h

```diff
@@ -157,6 +157,7 @@ CVMFS_EXPORT std::string GetArch();
 
 CVMFS_EXPORT int SetLimitNoFile(unsigned limit_nofile);
 CVMFS_EXPORT void GetLimitNoFile(unsigned *soft_limit, unsigned *hard_limit);
+CVMFS_EXPORT int SetLimitCore(unsigned limit_core);
```

Comment:
We should try adding a unit test for this.

Reply:
There aren't other unit tests for posix.cc functions so it doesn't seem like much of a priority to  me.